src/fmu/tools/rms/volumetrics.py

```diff
@@ -55,7 +53,7 @@ def myregionrenamer(s):
     """
     # First find out which row the data starts at:
     headerline = 0  # 0 is the first line
-    with open(txtfile) as volfile:
+    with open(str(txtfile)) as volfile:
```

Comment:
Do we need to convert to `str`? `open` supports Path objects directly https://docs.python.org/3/library/os.html#os.PathLike

```suggestion
    with open(txtfile) as volfile:
```

Reply:
Agree partially with Hans, but rather:
```
with txtfile.open() as volfile:
```

https://docs.python.org/3/library/pathlib.html#pathlib.Path.open

Reply:
`volumetrics.py` now also ported to `pathlib`